theme/templates/donate.html

```diff
@@ -0,0 +1,151 @@
+                <option value="IDR">IDR</option>
+                <option value="ILS">ILS</option>
+                <option value="INR">INR</option>
+                <option value="JPY">JPY</option>
+                <option value="KRW">KRW</option>
+                <option value="LTL">LTL</option>
+                <option value="LVL">LVL</option>
+                <option value="MXN">MXN</option>
+                <option value="MYR">MYR</option>
+                <option value="NOK">NOK</option>
+                <option value="NZD">NZD</option>
+                <option value="PHP">PHP</option>
+                <option value="PLN">PLN</option>
+                <option value="RON">RON</option>
+                <option value="RUB">RUB</option>
+                <option value="SEK">SEK</option>
+                <option value="SGD">SGD</option>
+                <option value="THB">THB</option>
+                <option value="TRY">TRY</option>
+                <option value="ZAR">ZAR</option>
+              </select>
+              <input type="hidden" name="data" value="DTV4JaKv709xq02lnDoV6YQzckpuFjSKkWgOIEaHst13ciMsAsGhH+lsBgTnSgZUhdcrDUWYx6UE4QolrN26/QN7O/4kpMfJkjkImN+nLRGmR05AM/UJhWVXsidBuo4N1u8LGqe1e5ohy+w7uRNew7y57qR8N6dngxEPcRHlPK9UuJu6OS3/aRho33vG0D0oVDUL4bwPTn5ITu1Eal1vM15fb/wWtoPX5+JwZgaYj2DdZJMEEV5T1K06zBhcAGWg">
+              <input name="submit" class="btn btn-default" src="//bitpay.com/img/donate-sm.png" type="image" alt="BitPay, the easy way to pay with bitcoins." border="0">
+            </form>
+          </div>
+        </div>
+        <div class="tab-pane" id="Flattr">
+          <h3 class="text-center">Via Flattr:</h3>
+          <div class="col-md-6 col-md-offset-3 text-center">
+            <a href="http://flattr.com/thing/1093304/Tryton" target="_blank">
+              <img src="//api.flattr.com/button/flattr-badge-large.png" alt="Flattr this" title="Flattr this" border="0">
+            </a>
+          </div>
+        </div>
+        <div class="tab-pane" id="gittip">
+          <h3 class="text-center">Via Gittip:</h3>
+          <div class="col-md-6 col-md-offset-3 text-center">
+            <iframe style="border: 0; margin: 0; padding: 0;" src="https://www.gittip.com/tryton/widget.html" width="78pt" height="55pt"></iframe>
+          </div>
+        </div>
+      </div>
+    </div>
+    <p class="text-center mailing-list">{% trans %}*If you want your donation to stay anonymous please say it so in donation comments{% endtrans %}</p>
+    <h2 class="text-center heading">{% trans %}Donors{% endtrans %}</h2>
+    <p class="text-center">{% trans %}The tryton Foundation wants to thank individuals and companies that finance it's activity. Donors are lsited in descending order using an algorithm that takes into account both the amount and the age of the donation over the last year.{% endtrans %}</p>
+    <br>
+    1. gcoop - Cooperative de Software Libre<br>
+    2. James Norman<br>
+    3. Mars Guitars <br>
```

Comment:
This list is dynamically generated.
Using this resource: http://www.tryton.org//assets/donators.txt
You can find the JS that fill it: http://www.tryton.org/foundation/donations.txt